feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenUI.kt

```diff
@@ -1,20 +1,93 @@
 package com.epmedu.animeal.tabs.more.about
 
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.layout.Arrangement.Absolute.SpaceEvenly
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import com.epmedu.animeal.foundation.placeholder.ScreenPlaceholder
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epmedu.animeal.foundation.button.AnimealSocialButton
 import com.epmedu.animeal.foundation.preview.AnimealPreview
 import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.topbar.BackButton
+import com.epmedu.animeal.foundation.topbar.TopBar
 import com.epmedu.animeal.resources.R
 
 @Composable
 internal fun AboutScreenUI(
+    modifier: Modifier = Modifier,
     onBack: () -> Unit,
+    onSocialFacebookClick: () -> Unit,
+    onSocialInstagramClick: () -> Unit,
+    onSocialLinkedinClick: () -> Unit,
+    onSocialWebClick: () -> Unit,
 ) {
-    ScreenPlaceholder(
-        title = stringResource(id = R.string.page_about),
-        onBack = onBack,
-    )
+    Column(
```

Comment:
Use `Scaffold` as a root with `topBar` so that inside it you can apply horizontal paddings on the whole column, not to each element